Invalid HTTP_HOST header: 'imperial.techevery.ng'. You may need to add 'imperial.techevery.ng' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/utils/deprecation.py", line 135, in __call__
    response = self.process_request(request)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/http/request.py", line 148, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'imperial.techevery.ng'. You may need to add 'imperial.techevery.ng' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'imperial.techevery.ng'. You may need to add 'imperial.techevery.ng' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/utils/deprecation.py", line 135, in __call__
    response = self.process_request(request)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/http/request.py", line 148, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'imperial.techevery.ng'. You may need to add 'imperial.techevery.ng' to ALLOWED_HOSTS.
Bad Request: /favicon.ico
Invalid HTTP_HOST header: 'imperial.techevery.ng'. You may need to add 'imperial.techevery.ng' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/utils/deprecation.py", line 135, in __call__
    response = self.process_request(request)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/http/request.py", line 148, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'imperial.techevery.ng'. You may need to add 'imperial.techevery.ng' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'imperial.techevery.ng'. You may need to add 'imperial.techevery.ng' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/utils/deprecation.py", line 135, in __call__
    response = self.process_request(request)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/http/request.py", line 148, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'imperial.techevery.ng'. You may need to add 'imperial.techevery.ng' to ALLOWED_HOSTS.
Bad Request: /favicon.ico
Invalid HTTP_HOST header: 'imperial.techevery.ng'. You may need to add 'imperial.techevery.ng' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/utils/deprecation.py", line 135, in __call__
    response = self.process_request(request)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/http/request.py", line 148, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'imperial.techevery.ng'. You may need to add 'imperial.techevery.ng' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'imperial.techevery.ng'. You may need to add 'imperial.techevery.ng' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/utils/deprecation.py", line 135, in __call__
    response = self.process_request(request)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/http/request.py", line 148, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'imperial.techevery.ng'. You may need to add 'imperial.techevery.ng' to ALLOWED_HOSTS.
Bad Request: /favicon.ico
Invalid HTTP_HOST header: 'imperial.techevery.ng'. You may need to add 'imperial.techevery.ng' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/utils/deprecation.py", line 135, in __call__
    response = self.process_request(request)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/http/request.py", line 148, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'imperial.techevery.ng'. You may need to add 'imperial.techevery.ng' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'imperial.techevery.ng'. You may need to add 'imperial.techevery.ng' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/utils/deprecation.py", line 135, in __call__
    response = self.process_request(request)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/http/request.py", line 148, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'imperial.techevery.ng'. You may need to add 'imperial.techevery.ng' to ALLOWED_HOSTS.
Bad Request: /favicon.ico
Not Found: /static/admin/css/fonts.css
Not Found: /favicon.ico
Not Found: /static/admin/css/fonts.css
Not Found: /static/admin/css/fonts.css
Invalid HTTP_HOST header: 'www.imperial.techevery.ng'. You may need to add 'www.imperial.techevery.ng' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/utils/deprecation.py", line 135, in __call__
    response = self.process_request(request)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/http/request.py", line 148, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.imperial.techevery.ng'. You may need to add 'www.imperial.techevery.ng' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'www.imperial.techevery.ng'. You may need to add 'www.imperial.techevery.ng' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/utils/deprecation.py", line 135, in __call__
    response = self.process_request(request)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/http/request.py", line 148, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.imperial.techevery.ng'. You may need to add 'www.imperial.techevery.ng' to ALLOWED_HOSTS.
Bad Request: /
Invalid HTTP_HOST header: 'www.imperial.techevery.ng'. You may need to add 'www.imperial.techevery.ng' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/utils/deprecation.py", line 135, in __call__
    response = self.process_request(request)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/http/request.py", line 148, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.imperial.techevery.ng'. You may need to add 'www.imperial.techevery.ng' to ALLOWED_HOSTS.
Bad Request: /
Not Found: /static/admin/css/fonts.css
Not Found: /static/admin/css/fonts.css
Invalid HTTP_HOST header: 'www.imperial.techevery.ng'. You may need to add 'www.imperial.techevery.ng' to ALLOWED_HOSTS.
Traceback (most recent call last):
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/utils/deprecation.py", line 135, in __call__
    response = self.process_request(request)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/middleware/common.py", line 48, in process_request
    host = request.get_host()
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/http/request.py", line 148, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: 'www.imperial.techevery.ng'. You may need to add 'www.imperial.techevery.ng' to ALLOWED_HOSTS.
Bad Request: /
Not Found: /static/admin/css/fonts.css
Not Found: /static/admin/css/fonts.css
Not Found: /favicon.ico
Not Found: /static/admin/css/fonts.css
Not Found: /static/admin/css/fonts.css
Not Found: /static/admin/css/fonts.css
Not Found: /favicon.ico
Not Found: /static/admin/css/fonts.css
Not Found: /robots.txt
Not Found: /static/admin/css/fonts.css
Not Found: /favicon.ico
Not Found: /robots.txt
Not Found: /favicon.ico
Not Found: /static/admin/css/fonts.css
Not Found: /static/admin/css/fonts.css
Not Found: /static/admin/css/fonts.css
Not Found: /static/admin/css/fonts.css
Not Found: /static/admin/css/fonts.css
Not Found: /favicon.ico
Not Found: /static/admin/css/base.css
Not Found: /static/admin/css/login.css
Not Found: /static/admin/js/nav_sidebar.js
Not Found: /static/admin/css/nav_sidebar.css
Not Found: /static/admin/css/responsive.css
Not Found: /static/admin/css/dark_mode.css
Traceback (most recent call last):
  File "/home/techwwpp/imperial-app-backend/passenger_wsgi.py", line 1, in <module>
    from config.wsgi import application
  File "/home/techwwpp/imperial-app-backend/config/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/apps/registry.py", line 91, in populate
    app_config = AppConfig.create(entry)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/apps/config.py", line 193, in create
    import_module(entry)
  File "/opt/alt/python38/lib64/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ModuleNotFoundError: No module named 'rest_framework'
Traceback (most recent call last):
  File "/home/techwwpp/imperial-app-backend/passenger_wsgi.py", line 1, in <module>
    from config.wsgi import application
  File "/home/techwwpp/imperial-app-backend/config/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/apps/registry.py", line 83, in populate
    raise RuntimeError("populate() isn't reentrant")
RuntimeError: populate() isn't reentrant
Traceback (most recent call last):
  File "/home/techwwpp/imperial-app-backend/passenger_wsgi.py", line 1, in <module>
    from config.wsgi import application
  File "/home/techwwpp/imperial-app-backend/config/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/apps/registry.py", line 83, in populate
    raise RuntimeError("populate() isn't reentrant")
RuntimeError: populate() isn't reentrant
Traceback (most recent call last):
  File "/home/techwwpp/imperial-app-backend/passenger_wsgi.py", line 1, in <module>
    from config.wsgi import application
  File "/home/techwwpp/imperial-app-backend/config/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/techwwpp/virtualenv/imperial-app-backend/3.8/lib/python3.8/site-packages/django/apps/registry.py", line 83, in populate
    raise RuntimeError("populate() isn't reentrant")
RuntimeError: populate() isn't reentrant
Not Found: /
Not Found: /static/admin/css/base.css
Not Found: /static/admin/css/login.css
Not Found: /static/admin/js/nav_sidebar.js
Not Found: /static/admin/css/nav_sidebar.css
Not Found: /static/admin/css/responsive.css
Not Found: /static/admin/css/dark_mode.css
Not Found: /account/api/register
Not Found: /account/api/register
Method Not Allowed: /accounts/api/register
Method Not Allowed: /accounts/api/register
Method Not Allowed: /accounts/api/register
Not Found: /favicon.ico
Not Found: /favicon.ico
Bad Request: /accounts/api/register
